cpp/include/kvikio/file_handle.hpp

```diff
@@ -502,45 +503,51 @@ class FileHandle {
    * This is an asynchronous version of `.read()`, which will be executed in sequence
    * for the specified stream.
    *
+   * When running CUDA v12.1 or older, this function falls back to use `.read()` after
+   * `stream` has been synchronized.
+   *
    * The arguments have the same meaning as in `.read()` but some of them are deferred. That is,
    * the values of `size`, `file_offset` and `devPtr_offset` will not be evaluated until execution
```

Comment:
```suggestion
   * the values pointed to by `size_p`, `file_offset_p` and `devPtr_offset_p` will not be evaluated until execution
```

This is not the case in fallback mode, right?